docusaurus/docs/guides/kubernetes/workload-tunneling/kubernetes-daemonset.md

```diff
@@ -10,7 +10,7 @@ This [daemonset manifest](https://get.openziti.io/tun/docker/ziti-tun-daemonset.
 
 The default nameserver address is `100.64.0.2`, but containers don't automatically use it until you configure cluster DNS. CoreDNS doesn't currently have a fallthrough mechanism, but you can use conventional names for your OpenZiti services' like `*.ziti`, and configure CoreDNS to forward queries that match that namespace to the OpenZiti nameserver.
 
-```yaml
```

Comment:
Are the yaml highlights actually creating a problem, or just bash? The yaml and json look right to me.